build.gradle

```diff
@@ -53,3 +56,174 @@ subprojects {
         }
     }
 }
+
+/**
+ * This task will copy all of the source code in the `library` module into the `library-v3` module
+ * and change its imports from the Maps SDK on Play Services to the Standalone V3 Beta Maps SDK.
+ */
+task copyGmsToV3Beta(type: Copy) {
+    group 'Custom'
+    description 'Generates source code for Maps SDK for Android V3 BETA Utilities'
+
+    from('library/src/main/java') {
+        filter { line ->
+            line.replace('com.google.android.gms.maps', 'com.google.android.libraries.maps')
+        }
+        into 'main/java'
+    }
+
+    from('library/src/main/res') {
+        into 'main/res'
```

Comment:
Could there be any cases here where we need to do replacements of import packages in layouts? I can't think of any, but just wanted to call this out.

Reply:
There weren't any gms specific imports in the layout file which is why I left out the package renaming filter. Think that should be added here (doesn't hurt)?

Reply:
I suppose it wouldn't hurt just in case things a future use case requires it. It might save someone in the future hunting down a bug due to mismatched package names.

Reply:
Makes sense. I'll add that here and add a comment why it was added.